Dockerfile.server

```diff
@@ -0,0 +1,13 @@
+FROM rust:latest as builder
+WORKDIR /usr/app/src
+COPY ./ /usr/app/src
+RUN apt-get update && apt-get -y install clang && \
+    rm -rf /var/lib/apt/lists/* && \
+    rustup component add rustfmt && \
```

Comment:
Does `rustfmt` need to be installed?

Reply:
If `rustfmt` is not in the image, the following error occurs when building

 ```
 => [builder 5/5] RUN --mount=type=cache,target=/usr/local/cargo/registry     --mount=type=cache,target=/rust/t  308.2s
 => => #   process didn't exit successfully: `/usr/app/src/target/release/build/indradb-proto-9d40d23f8a968169/build-sc
 => => # ript-build` (exit code: 1)
 => => #   --- stderr
 => => #   error: 'rustfmt' is not installed for the toolchain '1.51.0-x86_64-unknown-linux-gnu'
 => => #   To install, run `rustup component add rustfmt`
 => => # warning: build failed, waiting for other jobs to finish...
```